.changeset/curly-sheep-own.md

```diff
@@ -0,0 +1,8 @@
+---
+'@graphql-yoga/common': minor
+---
+
+Support `application/graphql`, `application/x-www-form-urlencoded` and `application/graphql+json` as defined in GraphQL over HTTP specification and implemented in `express-graphql` reference implementation so Yoga now accepts the following request bodies with specific "Content-Type" headers;
```

Comment:
Actually we already supported `application/graphql+json` since we didn't check the content-type in regular POST JSON parser. Now we do that by following the spec.